<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tw_detectors">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_manager">
      <attribute name="title">
       <string>Manager</string>
      </attribute>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>141</width>
         <height>107</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Manager</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="l_manager">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>not running</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>XRD1611:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="l_xrd1611">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>not connected</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>XRD1622:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="l_xrd1622">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>not connected</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>WidePIX:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="l_widepix">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>not connected</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Dexela:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="l_dexela">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>not connected</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="l_dummy">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Find Detectors</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
